admin/authentication/providers.md

```diff
@@ -0,0 +1,21 @@
+# Manage authentication providers
+
+Stoplight supports the following authentication providers:
+
+- Username and password
+- LDAP
+- <!-- QUERY: which others? -->
```

Comment:
We will continue to add to these providers.  There is nothing new this sprint, but we will add VCS SSO next.  Probably in July.